anda/lib/mesa/mesa.spec

```
@@ -72,14 +71,21 @@
 
 %global vulkan_drivers swrast%{?base_vulkan}%{?intel_platform_vulkan}%{?asahi_platform_vulkan}%{?extra_platform_vulkan}%{?with_nvk:,nouveau}%{?with_virtio:,virtio}%{?with_d3d12:,microsoft-experimental}
 
+%if 0%{?with_nvk} && 0%{?rhel}
+%global vendor_nvk_crates 1
+%endif
+
+# We've gotten a report that enabling LTO for mesa breaks some games. See
+# https://bugzilla.redhat.com/show_bug.cgi?id=1862771 for details.
+# Disable LTO for now
+%global _lto_cflags %nil
+
 Name:           %{srcname}
 Summary:        Mesa graphics libraries
-# Make the dep solver always prefer our Mesa over the distro's
-# This should not break anything by default as the Mesa stream is ***EXPLICITLY***
-# disabled by default, and has to be enabled manually. See `terra/release/terra-mesa.repo` for details.
-Epoch:          1
-Version:        25.3.0
-Release:        1%?dist
+%global ver 25.3.0
+Version:        %{lua:ver = string.gsub(rpm.expand("%{ver}"), "-", "~"); print(ver)}
```

While this is a change from Fedora the Epoch is necessary or the package will sort lower than the older ones.
```suggestion
Epoch:          1
Version:        %{lua:ver = string.gsub(rpm.expand("%{ver}"), "-", "~"); print(ver)}
```